exporter/tanzuobservabilityexporter/transformer_test.go

```diff
@@ -400,3 +400,43 @@ func spanWithStatus(statusCode ptrace.StatusCode, message string) ptrace.Span {
 	status.CopyTo(span.Status())
 	return span
 }
+
+func TestAttributesToTagsForMetrics(t *testing.T) {
+	// 1. Empty sourceKey: does not change resulting wfTags
+	attrMap := newMap(map[string]string{"k": "v"})
+	wfTags := attributesToTagsForMetrics("", attrMap)
+	assert.Equal(t, map[string]string{"k": "v"}, wfTags)
+
+	// 2. sourceKey exists in attrMap: delete from resulting wfTags
+	attrMap = newMap(map[string]string{"k": "v", "a_key": "a_val"})
+	wfTags = attributesToTagsForMetrics("a_key", attrMap)
+	assert.Equal(t, map[string]string{"k": "v"}, wfTags)
+
+	// 3. sourceKey is "source": delete from resulting wfTags. This scenario should only occur when
```

Comment:
I'm struggling to see the difference in behavior between scenario 3 and scenario 2. What is making `source` in scenario 3 any different from `a_key` in scneario 2?

Reply:
I had just moved this unit test to this file. But, you are right there is no difference. I added another assert to verify if the source key was deleted.